react_app/src/App.scss

```diff
@@ -0,0 +1,181 @@
+.App {
+  font-family: sans-serif;
+  text-align: center;
+}
+
+html {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+*, *:before, *:after {
```

Comment:
very nit: maybe just a freeline before you start here

Reply:
Done!